components/Button.js

```diff
@@ -0,0 +1,18 @@
+// create Button component
+import React, { Component } from 'react';
+// import view for creating and stylesheet for changing the appearance later
+import { View, StyleSheet } from 'react-native';
+
+function TBDmethod(){
+   // do whatever we want button to do     
+}
+
+export default class Butoon extends Component {
```

Comment:
lol butoon

Reply:
I would call it CustomButton or just call it button because it is a default export meaning when you import it, you can call it whatever you want.